maze_dataset/__init__.py

```diff
@@ -1,5 +1,9 @@
 from maze_dataset.constants import (
     SPECIAL_TOKENS,
+    VOCAB,
```

Comment:
it may make sense to put these in a different file -- i.e. something like `maze_dataset.tokenization.unified_vocab` or something